### ISSUE TITLE
GU0011 should not warn on fluent Moq

### ISSUE THREAD

**Issue by JohanLarsson** [GuOrg/Gu.Analyzers#142]
```cs
mock1.SetupGet(x => x.IsSatisfied).Returns(true)
```
